cross/librsync/Makefile

```diff
@@ -34,7 +34,7 @@ endif
 ifeq ($(findstring $(ARCH),$(x64_ARCHS)),$(ARCH))
 CMAKE_ARGS += -D CMAKE_SYSTEM_PROCESSOR=x86_64
 endif
-ifeq ($(findstring $(ARCH),$(i686_ARCES)),$(ARCH))
```

Comment:
sorry for this typo

Reply:
Well it's always easier to see others' ;)